gradle/init.gradle

```diff
@@ -85,11 +85,9 @@ if (isTopLevelBuild) {
             configureExtension(extensions["gradleEnterprise"])
         }
     } else {
-        //Versions less than 5 use the build scan plugin.
-        rootProject {
-            pluginManager.apply(com.gradle.scan.plugin.BuildScanPlugin)
-            configureExtension(extensions["com.gradle.build-scan"])
-        }
+        // Versions less than 5 can't publish scans to ge.openrewrite.org
```

Comment:
wait, I thought there was a way to enable this for gradle 4, we just were using the wrong task name?

Reply:
it doesn't work
see the error message